projects/storefrontlib/src/cms-structure/page/component/services/component-handler.service.ts

```diff
@@ -33,17 +33,9 @@ export class ComponentHandlerService {
    * @param componentMapping
    */
   protected resolve(componentMapping: CmsComponentMapping): ComponentHandler {
-    const matchedHandlers = (this.handlers ?? []).filter((handler) =>
-      handler.hasMatch(componentMapping)
-    );
-    if (matchedHandlers.length > 1) {
-      matchedHandlers.sort(
-        (a, b) =>
-          (a.getPriority ? a.getPriority() : 0) -
-          (b.getPriority ? b.getPriority() : 0)
-      );
-    }
-    if (isDevMode() && matchedHandlers.length === 0) {
+    const handler = resolveHandler(this.handlers, [componentMapping]);
```

Comment:
I guess we sneaked this in ;)

Reply:
yup, didn't want to create yet another PR just for this,